packages/itinerary-body/package.json

```diff
@@ -24,7 +24,7 @@
     "string-similarity": "^4.0.4"
   },
   "devDependencies": {
-    "@opentripplanner/types": "^6.1.0",
+    "@opentripplanner/types": "7.0.0-alpha.11",
```

Comment:
What is this change for?

Reply:
Adding `itinerary` to the Config type. It wasn't being recognized locally.

Reply:
Why isn't there a non-beta package that includes the definition?

Reply:
Instead of using a beta package, extend the Config type in the props type at `packages/itinerary-body/src/AccessLegBody/index.tsx` line 19 by writing something like:
```js
interface Props {
  config: Config & {
     itinerary?: {
       hideDrivingDirections?: boolean
     }
  }
  ...
}
```